### ISSUE TITLE
Errors in log when casting to a Chrome Cast group?!

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#77]
I started a Cast to a Chromecast group (everywhere) . That works!, But I noticed these loggings:
![image](https://user-images.githubusercontent.com/4363599/167289800-28972ff0-e2df-476b-b9bc-e51b77a09858.png)

```
2022-05-08 11:13:16 INFO (MainThread) [music_assistant.mass] Finished job [Analyze audio for spotify://track/6D1i1hoMyjs843XbMhNq8J] in 6.85 seconds.
2022-05-08 11:14:14 ERROR (MainThread) [frontend.js.latest.202205040] http://ha57:8123/lib/mass/mass.iife.js:54:128759 Uncaught TypeError: Cannot convert undefined or null to object
2022-05-08 11:14:14 ERROR (MainThread) [frontend.js.latest.202205040] http://ha57:8123/lib/mass/mass.iife.js:54:128759 Uncaught TypeError: Cannot convert undefined or null to object
2022-05-08 11:14:14 ERROR (MainThread) [frontend.js.latest.202205040] http://ha57:8123/lib/mass/mass.iife.js:54:128759 Uncaught TypeError: Cannot convert undefined or null to object
2022-05-08 11:14:14 ERROR (MainThread) [frontend.js.latest.202205040] http://ha57:8123/lib/mass/mass.iife.js:54:128759 Uncaught TypeError: Cannot convert undefined or null to object
2022-05-08 11:14:14 ERROR (MainThread) [frontend.js.latest.202205040] http://ha57:8123/lib/mass/mass.iife.js:54:128759 Uncaught TypeError: Cannot convert undefined or null to object
2022-05-08 11:14:14 ERROR (MainThread) [frontend.js.latest.202205040] http://ha57:8123/lib/mass/mass.iife.js:54:128759 Uncaught TypeError: Cannot convert undefined or null to object
2022-05-08 11:14:14 ERROR (MainThread) [frontend.js.latest.202205040] http://ha57:8123/lib/mass/mass.iife.js:54:128759 Uncaught TypeError: Cannot convert undefined or null to object
2022-05-08 11:14:14 ERROR (MainThread) [frontend.js.latest.202205040] http://ha57:8123/lib/mass/mass.iife.js:54:128759 Uncaught TypeError: Cannot convert undefined or null to object
2022-05-08 11:14:22 ERROR (Thread-13) [pychromecast.socket_client] [Everywhere_57(192.168.178.142):32034] Error reading from socket.
2022-05-08 11:14:22 WARNING (Thread-13) [pychromecast.socket_client] [Everywhere_57(192.168.178.142):32034] Error communicating with socket, resetting connection
2022-05-08 11:14:23 DEBUG (MainThread) [music_assistant.mass.stream] Finished Streaming queue track: ea837a5e-f12b-4c70-937a-073cb5e49e63 (Crimson Sun - Feed the Fire) on queue Audio57keuken
```

**Comment by marcelveldt** [comment]
The "reading from socket" errors are from Cast integration, not related to Music Assistant.
The other error is from the MA frontend, I'll need to take a look at it. It can't harm but it doesn't look very pretty

**Comment by marcelveldt** [comment]
I think I fixed the 2nd error in version 1.0.10. Please let me know if I missed a spot.

**Comment by erkr** [comment]
Haven't seen these logs with the latest version, will raise a new issue when needed 👍
